### DISCUSSION TITLE
Deactivate location (Latitude & Longitude) or set.location manually

### DISCUSSION THREAD

**Discussion by gruso70** [DarwinsBuddy/WienerNetzeSmartmeter#121] [Q&A]
I've installed the WienerNetzeSmartmeter with HomeAssistant on a NAS-Server and it works perfect. I would like to deactivate the location or change it manually but I don't know the entity, service or sensor.

service: sensor.at0010000000000000001000012345678.set_location
data:
  longitude: "12.345678"
  latitude: "12.345678"
  
I've tested the set_location for the sensor and there is an error for this handling.
Have any ideas for this?
Many thanks for help and ideas
Günter

**Comment by DarwinsBuddy** [comment]
Hi Günter,

I am not sure what you are trying to achieve.
Do you want to edit your Home Assistant installation's location?
This you can achieve via WebUI in Settings -> System -> General.
You can do the same via the service you've mentioned (`Home Assistant Core Integration: Set location`).

What this service is *not* for: Changing a sensor's/device's/entity's location.

Why would you want to change the sensor's location (which is reported by Wiener Netze)?

**Reply by gruso70** [reply]
Hello Darwin,
I know how to change the location of Home Assistant.
However, I would like to disable or change the location of "Wiener Netze" so that it is not visible in the cloud.
After the location of "Wiener Netze" is reported, I would like to disable it to not show it in the cloud.

**Comment by DarwinsBuddy** [comment]
I don't know what you mean with "in the cloud". You said you are hosting home assistant on your NAS, right?

**Comment by gruso70** [comment]
That's right, but the NAS has connection to the cloud and I want to disable the display of the location in the HA of the NAS, so that it can not be transferred to the cloud.
It is actually only a matter of deactivating the reading and provision of the location in "Wiener Netze Smartmeter".

**Comment by DarwinsBuddy** [comment]
If this issue still persists please create an issue.
Cleaning up the discussions currently :)
